tests/assets/integration_test_setup.py

```diff
@@ -4,21 +4,46 @@
 import yaml
 import logging
 import os
+import glob
+from pathlib import Path
+from snowshu.adapters.source_adapters.snowflake_adapter import SnowflakeAdapter
+from snowshu.core.models.credentials import Credentials
+
+# raise NotImplementedError('THIS IS A WORK IN PROGRESS, DO NOT USE!')
```

Comment:
If this is still under construction, we should at least leave the `raise NotImplementedError` in the script.